### ISSUE TITLE
Loops within spec classes breaks IDE support

### ISSUE THREAD

**Issue by nikitabobko** [JetBrains/spek#815]
All I did is simple test:
```kotlin
object Foo : Spek({
  val data = listOf(
    listOf(2, 2, 4),
    listOf(2, 3, 5)
  )

  for (row in data) {
    describe("${row[0]} + ${row[1]}") {
      val actualSum by memoized { row[0] + row[1] }
      it("equals ${row[2]}") {
        assertEquals(row[2], actualSum)
      }
    }
  }

  describe("foo") {
    it("works!") {
      assertEquals(1, 1)
    }
  }
})
```
And there is no way to launch particular test easily in IntelliJ (only all tests):
![image](https://user-images.githubusercontent.com/20517828/66960661-a0230400-f075-11e9-8a14-625423472588.png)

I'd even happy to have workaround like [this](https://github.com/spekframework/spek/issues/519). But we don't have this as well.

Please fix this issue. It makes your testing framework not production ready

**Comment by nikitabobko** [comment]
Sometimes it works, sometimes it not:
![image](https://user-images.githubusercontent.com/20517828/66963420-5558ba80-f07c-11e9-82ee-37b0dda9fa34.png)
So unstable

I use latest stable IDEA: `IntelliJ IDEA 2019.2.3 #IU-192.6817.14`
With latest Spek plugin installed: `2.0.8-IJ2019.2`

**Comment by raniejade** [comment]
Closing as duplicate of #814

**Comment by nikitabobko** [comment]
I'd not say that it's duplication. Look at the latest screenshot. First `describe` in `MyEventSpek` goes without any string interpolation or loops. But even in spite of this there is no run button

**Comment by raniejade** [comment]
You are right, sorry! The problem here is that loops within your test breaks IDE support. Descriptions with interpolation not being supported is another issue, which is the duplicate of #814.
